src/amoc_api_test_handler.erl

```diff
@@ -11,34 +11,34 @@
 
 -type state() :: list().
 
--spec init(tuple(), cowboy:req(), state()) -> {upgrade, protocol, cowboy_rest}.
+-spec init(tuple(), cowboy_req:req(), state()) -> {upgrade, protocol, cowboy_rest}.
```

Comment:
Line 14 is too long: -spec init(tuple(), cowboy_req:req(), state()) -> {upgrade, protocol, cowboy_rest}..